_rules/lists-follow-content-model-a73be2.md

```diff
@@ -0,0 +1,135 @@
+---
+id: a73be2
+name: List elements follow content model
+rule_type: atomic
+description: |
+  This rule checks that list elements follow content model
```

Comment:
Maybe something like


  This rule checks that lists have one or more `li`, or `dd/dt` groups.

If I'm getting this right.